backends/platform/sdl/sdl.cpp

```diff
@@ -490,3 +490,15 @@ Common::TimerManager *OSystem_SDL::getTimerManager() {
 	return _timerManager;
 #endif
 }
+
+bool OSystem_SDL::hasFeature(Feature f) {
```

Comment:
Can you implement this function and below in SDL graphics backend? I would avoid changes here.